examples/using-jest/src/components/image.js

```diff
@@ -1,32 +1,33 @@
 import * as React from "react"
-import { StaticQuery, graphql } from "gatsby"
+import { useStaticQuery, graphql } from "gatsby"
```

Comment:
You can revert the `using-jest` changes as I changed that example in https://github.com/gatsbyjs/gatsby/pull/36763 (to be in line with docs changes)

Reply:
Done in https://github.com/gatsbyjs/gatsby/pull/36890/commits/ad063b434144c7f533088479827092ce7de6a833